compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -38,6 +38,7 @@ class CompilationTests {
       compileFilesInDir("tests/pos-custom-args/captures", defaultOptions.and("-language:experimental.captureChecking")),
       compileFile("tests/pos-special/utf8encoded.scala", defaultOptions.and("-encoding", "UTF8")),
       compileFile("tests/pos-special/utf16encoded.scala", defaultOptions.and("-encoding", "UTF16")),
+      compileDir("tests/pos-special/i18589", defaultOptions.and("-Ysafe-init").without("-Ycheck:all")),
```

Comment:
is there a reason for these special flags?

Reply:
What's the failure?

Reply:
Some positions in the tree are missing. 
```
*** error while checking tests/pos-special/i18589/test_1.scala after phase inlining ***
assertion failed: position not set for NamedCodecPlatform.Builder[Any] # -1
of class dotty.tools.dotc.ast.Trees$TypeTree in tests/pos-special/i18589/test_1.scala
```